src/_css/main.css

```diff
@@ -425,8 +425,8 @@ img {
 }
 
 dark-mode-toggle {
-  --dark-mode-toggle-light-icon: url(/_css/img/sun.svg);
-  --dark-mode-toggle-dark-icon: url(/_css/img/moon.svg);
+  --dark-mode-toggle-light-icon: url(/_css/img/moon.svg);
+  --dark-mode-toggle-dark-icon: url(/_css/img/sun.svg);
```

Comment:
lolwhoops

Reply:
Right? 

But in the end, it's this old discussion if a button should show the current state, or the state after pressing the button.

Reply:
Previous version (before this change) looks more intuitive to me 🙊

Reply:
If you look at the [demo](https://googlechromelabs.github.io/dark-mode-toggle/demo/index.html), the naming makes sense for the `switch`, but not so much for the `toggle` appearance. Would renaming to `--dark-mode-toggle-on-icon` and `--dark-mode-toggle-off-icon` make it any better?

Reply:
Oh, it seems that I was looking at old version. What I see now at [v8.dev](https://v8.dev) LGTM 🙂.